shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -0,0 +1,572 @@
+$T$ & 10\% & The Treasury Top Slice Percentage \\\hline
+$MER$ & 10\% &  The ``Monetary Expansion Rate'' per Year \\\hline
+$E$ & 1-5 &  Elapsed days per \cardano Epoch \\\hline
+\end{tabular}
+\end{center}
+\caption{Key Parameters that will be set by the community (initially determined by \IOHK).}
+\end{figure}
+
+\noindent
+Four key operating parameters are set by the community (the initial values will be determined by \IOHK):
+$k$, the target number of stakepools;
+$T$, the treasury top slice percentage;
+$MER$, the monetary expansion per year;
+and
+$E$, the length of each epoch, in days.
+%
+The target number of Stakepools is used to cap the rewards that any individual stakepool can receive. The intention is to encourage the creation of more stakepools, and to avoid domination by any single stake holder.
+The \emph{treasury top slice percentage} is the fraction of reward that is allocated to the treasury to cover fixed operating costs, and
+ensure the long-term viability of the \cardano{} system.  It is initially set to a small percentage (10\%) of the rewards.
+The \emph{monetary expansion rate} is the rate at which rewards are allocated from the \emph{reserves} of \ada{}.
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Value} & \textbf{Description} & \textbf{Calculated as} \\\hline
+$A_T$ & \ADA{45bn} & The total \ada{} that could ever be created & \\\hline
+\emph{P} & 20\%-50\% & Participation Rate in the Shelley Incentivised Testnet & \\\hline
+$A_c$ & \ADA{31bn} & The total \ada{} in circulation at Shelley launch & \\\hline
+$A_r$ & \ADA{14bn} & The total \ada{} in the reserves at Shelley launch & $A_T - A_c$ \\\hline
+\hline
+\end{tabular}
+\end{center}
+\caption{Parameters that are set by external factors (e.g. \ada{} holders).}
+\end{figure}
+
+\noindent
+Several parameters are pre-determined by external factors. These include the
+total \ada{} that could ever be created $A_T$;
+the total \ada{} that is in \emph{circulation} when the Shelley system launches
+(i.e. all \ada{} that is held by any entity on the launch date), $A_c$;
+and the \ada{} that is held in reserve when the system starts.
+These values are fixed and will not change.
+
+\clearpage
+\subsection{Stakepools}
+
+\subsubsection*{Operator-Set Parameters}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Range} & \textbf{Description} \\\hline
+$SP_c$ &  & Cost per day in \ada{} \\\hline
+$SP_m$ &  0\%-100\% & Percentage fee on rewards (``margin'') \\\hline
+\end{tabular}
+\end{center}
+\caption{Key Parameters that are set by the StakePool Operator.}
+\end{figure}
+
+\noindent
+The two main parameters that need to be set by the \emph{operator} are the cost per day that will be charged to
+the pool (in \ada), $SP_c$; and the percentage of any rewards that will subsequently be taken as a fee, $SP_m$.
```

Comment:
do we want per day here? isn't this more per epoch, e.g., per reward distribution?